OpenSim/Simulation/Test/testAssemblySolver.cpp

```diff
@@ -131,6 +131,8 @@ void testAssembleModelWithConstraints(string modelFile)
     for(int i=0; i< coords.getSize(); i++) {
         cout << "Coordinate " << coords[i].getName() << " get value = " << coords[i].getValue(state) << endl;
     }
+    auto coordsInOrder = model.getCoordinateNamesInMultibodyTreeOrder();
+    cout << coordsInOrder << std::endl;
```

Comment:
Consider adding a simple check that `coordsInOrder` has the same size as the `CoordinateSet`.

Reply:
Done